docs/user/agent/agent-services.md

```diff
@@ -0,0 +1,9 @@
+# Agent-based Installer Services
```

Comment:
It would be nice to add just a small introduction (few lines), just to describe what are these diagrams and in which moment of the lifecycle are triggered (the bootstrap)

Reply:
Good idea. Added.